mixins/exhibitionChapters.js

```diff
@@ -0,0 +1,49 @@
+export default {
+  methods: {
```

Comment:
Did you look into adding some unit tests for this mixin?
I think it would be useful to add tests as this code is going to be used in multiple locations.